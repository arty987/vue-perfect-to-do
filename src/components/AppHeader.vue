<script>
export default {
  name: "AppHeader"
}
</script>

<template>
  <header/>
</template>

<style scoped>
header {
  background-image: url('../assets/header.jpg');
  background-repeat: no-repeat;
  background-size: cover;
  background-position: center;
  width: 100%;
  height: 350px;
}
</style>