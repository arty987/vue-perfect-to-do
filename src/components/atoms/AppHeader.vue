<script>
export default {
  name: "AppHeader"
}
</script>

<template>
  <header/>
</template>

<style scoped lang="scss">
header {
  background-image: linear-gradient(70deg, rgba(128, 43, 193, 0.64) 30%, rgba(101, 162, 211, 0.65) 79%), url("../../assets/header.jpg");
  background-repeat: no-repeat;
  background-size: cover;
  background-position: center;
  width: 100%;
  height: 300px;
}
</style>