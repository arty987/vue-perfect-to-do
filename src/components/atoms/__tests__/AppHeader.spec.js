import {shallowMount} from '@vue/test-utils'
import AppHeader from '@/components/atoms/AppHeader.vue';

describe('AppHeader.vue', () => {
    it('Should render AppHeader', () => {
        const wrapper = shallowMount(AppHeader);
        expect(wrapper.element).toMatchSnapshot();
    });
})
