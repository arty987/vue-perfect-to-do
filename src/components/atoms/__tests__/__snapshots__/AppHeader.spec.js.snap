// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader.vue Should render AppHeader 1`] = `<header />`;
